.github/workflows/python-package.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Assuming it works on the APBS side, I'd recommend:

```
python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Note that the quotes are important to make sure "3.10" isn't interpreted as "3.1".

Reply:
Ah I gotcha. Just updated the workflow versions as well as the PDB2PQR version v3.6.1. Currently waiting for the build to finish